bombastic/walker/src/lib.rs

```diff
@@ -98,14 +100,18 @@ impl Run {
                     let keys = self
                         .signing_key
                         .into_iter()
-                        .chain(self.devmode.then(|| Url::parse(DEVMODE_KEY).unwrap()))
+                        .chain(
+                            self.devmode
+                                .then(|| Url::parse(DEVMODE_KEY).context("failed to parse devmode key"))
+                                .transpose()?,
+                        )
```

Comment:
Can this even happen? I like how this guy puts it: https://stackoverflow.com/a/39478185

Reply:
It should not. Then again, that's what engineers think about most of their bugs. The alternative would be to use `.expect("message")`. Which is basically `unwrap` but with a proper message.